### DISCUSSION TITLE
how to prevent keka from deleting archive after extraction and extract to a subfolder

### DISCUSSION THREAD

**Discussion by killtux** [aonez/Keka#1211] [Q&A]
i dont find the settings menu for unpacking (only the archiving options). where can i set that keka dont deletes the whole archive if i want to unpack it and extract the contents in a subfolder named like the archive file?

**Comment by gingerbeardman** [comment]
1. Open Keka
1. Keka from menu bar, then Preferences menu (or press <kbd>Cmd+,</kbd> to get there quicker)
1. Extraction tab:

<img width="624" alt="Screen shot 2023-01-29 at 11 39 51" src="https://user-images.githubusercontent.com/49612/215323696-4f86022f-e292-478e-97b9-805500218f6c.png">

**Reply by killtux** [reply]
wow thanks a lot. i did mistake the "settings" that pop up when starting keka directly for the actual settings menu.

**Reply by gingerbeardman** [reply]
No worries!
